modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running multiple applications side-by-side
+* Clear boundaries for operations on what a Mule application is
```

Comment:
This sentence is also unclear. Does this mean clear boundaries for running operations?

Reply:
@aiannucci Could you reconfirm?

Reply:
Updated.